.devcontainer/create-db-user.sql

```diff
@@ -0,0 +1,2 @@
+CREATE USER vscode CREATEDB;
+CREATE DATABASE vscode WITH OWNER vscode;
```

Comment:
does this mean this setup only works with vscode? not other IDEs?

Reply:
Seems like devcontainers is a pretty cool VSCode [extension](https://code.visualstudio.com/docs/devcontainers/create-dev-container). VSC just keeps getting cooler and cooler. 💎

Reply:
Yeah. As far as I can tell, this is a vscode-only thing. However, the dockerfile seems to not be vscode-specific, so it could probably be reused in other situations. (That said, it works pretty well with VS Code.)